### PR TITLE
Addition of a recipe for the Anesthesia kit

### PR DESCRIPTION
**Pull request by Alexandreiah**
**What does it do?**
This PR introduces a recipe a way to obtain anesthesia kit in a balanced way

**Rationale**
So at first this recipe used painkillers but that wasnt a real way to get someone to sleep (at least not without killing them) so with this PR and after some discussion on the DDA discord I added a way to craft it by using Nitrous oxide (NO2), since that chemical is used regularly as a way to provide anesthetic, and since the easier and more direct way to do it is kind of dangerous (and we have no way to add possible explosions to crafting) this is what it!

### CODE REVIEW

**Comment by Xhuis** [comment]
I'm overall very supportive of this change. Once Augmentation Associate was added in, I began playtesting the new changes and bionics, and my current view on it is quite negative, but that's a lot of reasons that I won't pour out into a novel here.

Something to consider - Always treat the skill requirement on recipes as the point at which they are post-scarce. In this case, cooking and first aid 6 is when anesthesia should be considered much less valuable by virtue of being obtainable with raw materials, Make sure that that's the point at which you want it to do so!

**Comment by Mecares** [comment]
That sounds extremely dangerous. I am not sure if this does not go too far for the maingame. Maybe better suited for modcontent?

**Comment by Night-Pryanik** [comment]
Maybe rename this variant as a "makeshift anesthetic"? 
The "normal" anesthesia kits are highly effective intravenous-administered drugs. I'd say Nitrous oxide is not that effective, and autodoc could consume two charges of this makeshift anesthetic instead of one charge for "normal" one?

**Comment by cainiaowu** [comment]
Also there will be a heavy debuff after inhaling nitrous oxide.

https://toxnet.nlm.nih.gov/cgi-bin/sis/search/a?dbs+hsdb:@term+@DOCNO+504

"The main complication following inhalation of nitrous oxide is varying degrees of hypoxia, affecting the functions of the heart and the brain. This may be associated with hypotension, fatal cardiac arrhythmias, headache, dizziness, anoxic brain damage, cerebral edema and permanent mental deficit. "

**Comment by Firestorm01X2** [comment]
>That sounds extremely dangerous. I am not sure if this does not go too far for the maingame. Maybe better suited for modcontent?

So drinking mutagens should be. No. Just no.

**Comment by DracoGriffin** [comment]
>Also there will be a heavy debuff after inhaling nitrous oxide.
https://toxnet.nlm.nih.gov/cgi-bin/sis/search/a?dbs+hsdb:@term+@DOCNO+504
"The main complication following inhalation of nitrous oxide is varying degrees of hypoxia, affecting the functions of the heart and the brain. This may be associated with hypotension, fatal cardiac arrhythmias, headache, dizziness, anoxic brain damage, cerebral edema and permanent mental deficit. "

This is a good point. However, I would like to believe the AutoDoc manages the care for this as any anesthesiologist would, and there shouldn't be any Bad Things™ that happen, but having an aftereffect for using "makeshift" anesthesia (maybe the AutoDoc isn't calibrated correctly for it) that can last for hour or several (shouldn't be more than six). Temporary and minor stat loss with maybe some speed reductions.

**Comment by Night-Pryanik** [comment]
> Temporary and minor stat loss with maybe some speed reductions.

Also vertigo and weakness.

File: data/json/recipes/recipe_medsandchemicals.json [hunk 1]
```diff
@@ -2395,5 +2395,29 @@
...
+	"type": "recipe",
+	"category": "CC_OTHER",
+	"subcategory": "CC_OTHER_MEDICAL",
+	"skill_used": "cooking",
+	"difficulty": 6,
+	"skills_required": [ "firstaid", 6 ],
+	"time": 40000,
+	"book_learn":[ [ "textbook_firstaid", 6 ], [ "emergency_book", 5 ] ],
+	"qualities": [
+		{ "id": "CHEM", "level": 3},
```
[the remark this answers is not in the document]
Comment (Alexandreiah) [reply data/json/recipes/recipe_medsandchemicals.json]: Done

**Comment by John-Candlebury** [comment]
No a real fan of this change, It really trivializes any challenge/risk anesthesia is supposed to bring to the game.

Maybe consider making the recipe require something common but hard to obtain. Like royal jelly/marloss berries/fungaloid or triffid parts/razorclaw roe?

**Comment by Lasiace** [comment]
To be fair, bismuth is a required ingredient and is pretty rare. You can only get bismuth from, er... professional fishing rods?

...Where do you actually _get_ bismuth?

**Comment by Ker-Nes** [comment]
The hard part of anesthesia is controlling the dosage and monitoring the condition, not really getting the anesthetic in question. The player can already make opioids (including heroin, which works very similarly to morphine), it's no great jump for them to pack all together in a format the autodoc can use it. 

Yes, those are very dangerous, specially when used for general anesthesia, but anything significantly weaker is not going to cut for the purposes that they're needed. NO2 alone is not used for any major surgery, but as support to other stronger drugs.

If we are to need an specific anesthesia kit for the autodoc, and not merely ingesting enough painkillers beforehand and overriding the autodoc when it lacks purpose built anesthesia, we need to assume that the autodoc is capable of some degree of monitoring the character's condition and automatic controlling the dosage.  

That is, that the autodoc, with the correct materials and instructions, can handle the putting the knock out and then back up again afterwards, which means that we could safely assume that they're using the big drugs, not just NO2.

In all, I heavily support the addition of a recipe, but I'd have it using both the current ingredients and some stronger painkiller (maybe the player choice's of any of the heavier ones in different dosages according to their strength). Furthermore, for further development of the CBM install system, I'd say that it might be interesting to either have an extra first aid skill check, for setting up the instructions for the anesthesia and patient information, or just assume it is present in the one already existing.

**Comment by sfsworms** [comment]
Someone had a nice proposal for a recipe for anesthetic in the original PR or somewhere (something with swamp gas and some natural product extracted from berries), but my GitHub-fu is weak and I can't seem to find it.

**Comment by BrianLefler** [comment]
https://www.quora.com/How-do-you-make-sodium-pentothal

- sulfur +hydrogen => hydrogen sulfide (sewer gas) [at 450 °C]
- hydrogen sulfide (sewer gas) + calcium cyanamide (fertilizer) => thiourea
- grapes or strawberries => diethyl malonate [via purification]
- thiourea + diethyl malonate => sodium pentothal

I think Sodium Pentothal (also called truth serum) was the main anesthetic in regular use but has now been replaced by Propofol.  I don't know why.  So you could call it "homemade anesthetic" (as inferior to propofol) or use this as the main recipe for anesthetic (which it was for decades), whichever you prefer.

So a recipe could be one step:

- sulfur + hydrogen + fertilizer + grapes/strawberries = homemade anesthetic

Or two if you want to specifically call out swamp gas and allow other sources for it:

- sulfur + hydrogen = swamp gas
- swamp gas + fertilizer + grapes/strawberries = homemade anesthetic

I don't know what quantities you'd want to use for the reagents. Either way the recipes would require a chemistry lab.

**Comment by kevingranade** [comment]
As far as I can tell, this is accurate:
> NO2 alone is not used for any major surgery, but as support to other stronger drugs.

NO2 can be the initial knockout agent, but needs a stronger agent to maintain anesthesia.
Mentioned alternatives are, [Sevoflurane](https://en.wikipedia.org/wiki/Sevoflurane), [Desflurane](https://en.wikipedia.org/wiki/Desflurane) and [Isoflurane](https://en.wikipedia.org/wiki/Isoflurane), and [Sodium pentothal](https://en.wikipedia.org/wiki/Sodium_thiopental).

Sodium Pentothal is actually really promising since it seems to be the only chemical you need to induce and maintain general anesthesia.

**Comment by sfsworms** [comment]
Ether would probably be the best, easiest bet for a survivor-made anesthesic, with the side effect of very bad nausea and vomiting. (Not sure about inhaling, but a friend of mine that drank some once described it as "the worst hangover ever, time two", and it fits with the description of side effect for surgery.)

It could be produced via the acid ether synthesis (fairly straightforward: heat ethanol and suplhiric acid and distill out the ether), although some other recipes are possible. At a slightly higher cooking level, dehydration of ethanol with aluminum oxide as a catalyst should be doable.

**Comment by Ker-Nes** [comment]
>Ether would probably be the best, easiest bet for a survivor-made anesthesic, with the side effect of very bad nausea and vomiting. (Not sure about inhaling, but a friend of mine that drank some once described it as "the worst hangover ever, time two", and it fits with the description of side effect for surgery.)
>
>It could be produced via the acid ether synthesis (fairly straightforward: heat ethanol and suplhiric acid and distill out the ether), although some other recipes are possible. At a slightly higher cooking level, dehydration of ethanol with aluminum oxide as a catalyst should be doable.

Agreed. I'd put poppy painkillers at the lowest (it's likely essentially roughly refined morphine plus other opioids), which nonetheless should be *effective* for at least simpler surgeries, if addicting and dangerous.

Speaking of which, might be interesting to, eventually, set different "rates" of CBM install invasiveness (and potential pain) to combine with different rates of painkillers. You can, say, install the alarm system with some pretty low grade and weak anesthetics, but things like the cerebral booster would take only the best and most effective of them?

**Comment by kevingranade** [comment]
Sodium Penethol or Ether would be great, but NO2 doesn't work.
NO2 is potentially useful for other stuff though, so the recipe could still go in with NO2 having different effects.

**Pull request closed without merging**